src/loaders/svg/tvgSvgLoader.cpp

```diff
@@ -1125,12 +1125,28 @@ static bool _parseStyleAttr(void* data, const char* key, const char* value, bool
     sz = strlen(key);
     for (unsigned int i = 0; i < sizeof(styleTags) / sizeof(styleTags[0]); i++) {
         if (styleTags[i].sz - 1 == sz && !strncmp(styleTags[i].tag, key, sz)) {
+            string newValue = value;
```

Comment:
No many beneficial using string here. C style uses less bin size. Also don't need to copy value here.